transform/models/staging/dof_mdsa/stg_footprints_block.sql

```diff
@@ -0,0 +1,13 @@
+with footprints_block as (
+    select 
+f."release",
+f."capture_dates_range",
+f."geometry",
+b."GEOID20",
+b."NAME20"
```

Comment:
Here and below, I'd suggest giving these better names

Reply:
Makes sense